referrals/router.py

```
@@ -0,0 +1,104 @@
+from fastapi import APIRouter, HTTPException, Body, Depends, Path, Query
```

Ruff (0.11.9)

`fastapi.Body` imported but unused

Remove unused import

(F401)